lib/hydra_editor/version.rb

```diff
@@ -1,3 +1,3 @@
 module HydraEditor
-  VERSION = '4.0.2'.freeze
+  VERSION = '5.0.0'.freeze
```

Comment:
I don't understand why code climate isn't marking this as covered after I added the test for version.  I think this is the only changed line that is included in the diff coverage (currently at 0%).